test/integration-test.py

```diff
@@ -83,6 +81,80 @@ def expect(target_time, num, table):
     expect(now, 0, "authz")
     expect(after_grace_period, 1, "authz")
 
+def run_janitor():
+    # Set the fake clock to a year in the future such that all of the database
+    # rows created during the integration tests are older than the grace period.
+    now = datetime.datetime.utcnow()
+    target_time = now+datetime.timedelta(days=+365)
+
+    e = os.environ.copy()
+    e.setdefault("GORACE", "halt_on_error=1")
+    e.setdefault("FAKECLOCK", fakeclock(target_time))
+
+    # Note: Must use exec here so that killing this process kills the command.
+    cmdline = "exec ./bin/boulder-janitor --config test/config-next/janitor.json"
+    p = subprocess.Popen(cmdline, shell=True, env=e)
+    p.cmd = cmdline
```

Comment:
We assign to `p.cmd` in startservers so if `check()` shows an error we can say which process had the error. Here we should probably leave it out since it doesn't do anything.

The alternate would be to append `p` to `startservers` global `processes` list, but I'm reluctant to do that because it notionally belongs to `startservers` (and we don't do that for the `expired-authz-purger`).

Reply:
Makes sense, fixed in 01dd399